docs/sidebars.js

```diff
@@ -81,17 +81,17 @@ module.exports = {
     {
       "Flashbots Protect": [
         'flashbots-protect/overview',
+        'flashbots-protect/quick-start',
+        'flashbots-protect/mev-share',
+        'flashbots-protect/key_considerations',
+        'flashbots-protect/cancellations',
+        'flashbots-protect/stuck_transactions',
         {
-          'RPC': [
-            'flashbots-protect/rpc/quick-start',
-            'flashbots-protect/rpc/mev-share',
-            'flashbots-protect/rpc/uncle-bandits',
```

Comment:
Why did we get rid of the page on uncling? I also saw we don't mention uncled blocks as a risk in the overview anymore

Reply:
The page is outdated as there are no uncle blocks in PoS Ethereum.